typings/tsd.d.ts

```diff
@@ -1,7 +1,7 @@
-/// <reference path="../node_modules/immutable/dist/immutable.d.ts" />
-/// <reference path="../node_modules/reflect-metadata/reflect-metadata.d.ts" />
 /// <reference path="chrome/chrome.d.ts" />
 /// <reference path="es6-promise/es6-promise.d.ts" />
 /// <reference path="filesystem/filesystem.d.ts" />
 /// <reference path="filewriter/filewriter.d.ts" />
 /// <reference path="webrtc/MediaStream.d.ts" />
+/// <reference path="../node_modules/immutable/dist/immutable.d.ts" />
```

Comment:
I tried to get rid of these, but unfortunately we do. In short these typings were written in a way that does not seem to play well with this approach. I think that once immutable is updated we can get rid of these.